libraries/standard/mqtt/include/mqtt.h

```diff
@@ -406,6 +406,26 @@ MQTTStatus_t MQTT_ReceiveLoop( MQTTContext_t * pContext,
  */
 uint16_t MQTT_GetPacketId( MQTTContext_t * pContext );
 
+/**
+ * @brief Parses the payload of an MQTT SUBACK packet that contains the status
+ * code of subscription request made for topic filters in the equivalent
+ * subscribe packet.
```

Comment:
Nit: status code should be plural:
```suggestion
 * @brief Parses the payload of an MQTT SUBACK packet that contains status codes
 * corresponding to topic filter subscription requests from the original
 * subscribe packet.
```